doc/user_guide/marks/geoshape.rst

```diff
@@ -0,0 +1,689 @@
+This means that if your source data contain (multi)lines, you will have to explicitly define the ``filled`` value as ``False``.
+
+Compare:
+
+.. altair-plot::
+    
+    gs_line = gpd.GeoSeries.from_wkt(['LINESTRING (0 0, 1 1, 0 2, 2 2, -1 1, 1 0)'])
+    alt.Chart(gs_line).mark_geoshape().project(
+        type='identity', 
+        reflectY=True
+    )    
+
+With:
+
+.. altair-plot::
+    
+    gs_line = gpd.GeoSeries.from_wkt(['LINESTRING (0 0, 1 1, 0 2, 2 2, -1 1, 1 0)'])
+    alt.Chart(gs_line).mark_geoshape(
+        filled=False
+    ).project(
+        type='identity', 
+        reflectY=True
+    ) 
+
+Using this approach one can also style Polygons as if they are Linestrings:
+
+.. altair-plot::
+
+    alt.Chart(gdf_sel).mark_geoshape(
+        filled=False,
+        strokeWidth=1.5
+    ).encode(
+        stroke='name:N'
+    )
+
+Mapping Points
+~~~~~~~~~~~~~~
+Points can be drawn when they are defined as ``Points`` within a GeoDataFrame using ``mark_geoshape``.
+We first assign the centroids of Polygons as Point geometry and plot these:
+
+.. altair-plot::
+
+    gdf_centroid = gpd.GeoDataFrame(
+        data=gdf_sel.drop('geometry', axis=1), 
+        geometry=gdf_sel.geometry.centroid
+    )
+
+    alt.Chart(gdf_centroid).mark_geoshape()
+
+
+Caveat: To use the ``size`` encoding for the Points you will need to use the ``mark_circle`` in combination with the ``latitude`` and ``longitude`` encoding channel definitions.
+
+.. altair-plot::
+
+    gdf_centroid['lon'] = gdf_centroid['geometry'].x
+    gdf_centroid['lat'] = gdf_centroid['geometry'].y
+
+    alt.Chart(gdf_centroid).mark_circle().encode(
+        longitude='lon:Q',
+        latitude='lat:Q',
+        size="pop_est:Q"
+    )
+
+You could skip the extra assignment to the ``lon`` and ``lat`` column in the GeoDataFrame and use the coordinates directly.  We combine the chart with a basemap to bring some perspective to the points:
+
+.. altair-plot::
+
+    basemap = alt.Chart(gdf_sel).mark_geoshape(
+        fill='lightgray', stroke='white', strokeWidth=0.5
+    )
+
+    bubbles = alt.Chart(gdf_centroid).mark_circle(
+        stroke='black'
+    ).encode(
+        longitude='geometry.coordinates[0]:Q',    
+        latitude='geometry.coordinates[1]:Q',
+        size="pop_est:Q" 
+    )
+
+    (basemap + bubbles).project(
+        type='identity', 
+        reflectY=True
+    )
+
+Altair also contains expressions related to geographical features. We can for example define the ``centroids`` using a ``geoCentroid`` expression:
+
+.. altair-plot::
+
+    from altair.expr import datum, geoCentroid
+
+    bubbles = alt.Chart(gdf_sel).transform_calculate(
+        centroid=geoCentroid(None, datum)
+    ).mark_circle(
+        stroke='black'
+    ).encode(
+        longitude='centroid[0]:Q',
+        latitude='centroid[1]:Q',
+        size="pop_est:Q" 
+    )
+
+    (basemap + bubbles).project(
+        type='identity', reflectY=True
+    )
+
```

Comment:
```suggestion
Choropleths
~~~~~~~~~

An alternative to showing the population sizes as bubbles, is to create a "Choropleth" map. These are geographical heatmaps where the color or each region are mapped to the values of a column in the dataframe. 

.. altair-plot::

    alt.Chart(gdf_sel).mark_geoshape().encode(
        color='pop_est'
    )

When we create choropleth maps, we need to be careful, because although the color changes according to the value of the column we are interested in, the size is tied to the area of each country and we might miss interesting values in small countries just because we can't easily see them on the map (e.g. if we were to visualize population density).
```

What do you think of adding a section like this to introduce choropleths before getting into the classification section further down?